php/Terminus/Collections/Instruments.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Terminus\Collections;
+use Terminus\Request;
+use Terminus\User;
+use Terminus\Instrument;
+use \Terminus_Command;
+
+class Instruments {
+  private $_user;
```

Comment:
Can we change our styleguide to not require underscores for private properties and methods? I think the `private` declaration is "good enough" and I just really hate parsing this syntax `->_`.